app/main.py

```diff
@@ -16,4 +16,8 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [my_favourite_films, profile_info, marks],
```

Comment:
The variable `profile_info` is a tuple, which is immutable. It should be placed in the 'immutable' list instead of 'mutable'.